assignment1.txt

```diff
@@ -0,0 +1,7 @@
+Drew French
+
```

Comment:
where's your answer to number 1? you also didn't explain the difference between a compile and run time error.